added two test strings to en.yml file

ErinClaudio opened the pull request:
### What github issue is this PR for, if any?
WIP #1257

### What changed, and why?
I added two text strings and commented in the en.yml file. I want to make sure I am using the proper nesting conventions before continuing. 


### How will this affect user permissions?
N/A

### How is this tested? (please write tests!) 💖💪
N/A

### Screenshots please :)
![Screen Shot 2021-01-14 at 3 04 57 PM](https://user-images.githubusercontent.com/20326770/104643199-0c762400-567a-11eb-8218-23e1de544aa7.png)



### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
How to embed:
`![alt text](https://media.giphy.com/media/1nP7ThJFes5pgXKUNf/giphy.gif)`

compwron commented on config/locales/en.yml:
```diff
@@ -57,3 +57,11 @@ en:
+        title: "New Judge"
+  page:
+    judge:
+      #edit:
```
leave out `edit`
indentation matters :)

compwron commented:
this is looking promising! :)

littleforest commented on config/locales/en.yml:
```diff
@@ -57,3 +57,11 @@ en:
+  page:
+    judge:
+      #edit:
+        title: "Edit Judge
```
@ErinClaudio thanks for working on this! I would start with defining the active record model and attributes for Judge according to [this documentation](https://guides.rubyonrails.org/i18n.html#translations-for-active-record-models). For example:

```
en:
  activerecord:
    models:
      judge:
        one: Judge
        other: Judges
    attributes:
      judge:
        name: Name
```


Once that is in place, you could create entries for new, edit, delete, etc. like so:

```
en:
  new_record: "New %{model}"
  edit_record: "Edit %{model}"
  destroy_record: "Delete %{model}"
```

And then call it like so:

```
@judge = Judge.first
model_name = @judge.class.model_name.human
new_judge_str = I18n.t(:new_record, model: model_name)
```

ErinClaudio replied:
Thank you for the documentation!

compwron commented:
This is looking promising! The build seems upset and I'm not sure why... I will try to take a look later today but feel free to beat me to it

ErinClaudio commented:
> This is looking promising! The build seems upset and I'm not sure why... I will try to take a look later today but feel free to beat me to it.

- one error- I missed a closing quote. 
- and I believe I cannot nest with repeating keys as I did below
`page:
    judge:
      title: "New Judge"
  page:
    judge:
      title: "Edit Judge"
      `

ErinClaudio commented:
Hello,
I have attempted to clean this up.

- lines 52 - 55

		I added `judge: new_title: "New Judge"` and `judge: edit_title: "Edit Judge."` Thinking that those key values should be under `page:` (line 48)

- line 73 and 75

		I added create and update keys.

- line 77 - 82 

I changed the keys to (79)`new_title:` and (82) `edit_title: `. The original repeating key might have been causing a failure. Please let me know which of the commented out lines I should remove. I look forward to your feedback and improvement.

compwron commented on config/locales/en.yml:
```diff
@@ -57,3 +61,22 @@ en:
+    models:
+      judge:
+        one: Judge
+        other: Judges
```
Let's leave out `other:` and use https://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize in the code instead

compwron commented on config/locales/en.yml:
```diff
@@ -57,3 +61,22 @@ en:
+  page:
+    judge:
+      new_title: "New Judge"
+  page:
```
YAML doesn't allow repeated keys like this- see https://stackoverflow.com/questions/47668308/duplicate-key-in-yaml-configuaration-file for an example

scottolsen replied:
For views you can also do something like
```yml
# config/locals/en.yml
judges:
  new:
    title: 'New Judge'
  edit:
    title: 'Edit Judge'
```

And in the view you can just reference it like below.
```ruby
# app/views/judges/edit.html.erb
<%= render partial: "form", locals: {title: t('.title'), judge: @judge} %>

# app/views/judges/new.html.erb
<%= render partial: "form", locals: {title: t('.title'), judge: @judge} %>
```

More info [here](https://guides.rubyonrails.org/i18n.html#lazy-lookup).

compwron commented:
Ping me in slack if you want to pair on this- I'm around a lot today and I can make time this week :)

compwron commented:
Heck yes!!
